waddrmgr/manager.go

```diff
@@ -1580,7 +1580,11 @@ func (m *Manager) LastExternalAddress(account uint32) (ManagedAddress, error) {
 		return nil, err
 	}
 
-	return acctInfo.lastExternalAddr, nil
+	if acctInfo.nextExternalIndex > 0 {
+		return acctInfo.lastExternalAddr, nil
+	}
+
+	return nil, managerError(ErrAddressNotFound, "address not found", nil)
```

Comment:
Since this isn't looking up a particular address by string, hash160, etc., can you change the error description string to something like "no previous external address"?

Also, need a similar error for LastInternalAddress below.